docs/docs/policies/actions.md

```diff
@@ -22,26 +22,98 @@ rules:
   - event: dropped_executable
 ```
 
-### Webhook action
+if we start tracee it will log table to stdout:
+
+```
```

Comment:
I don't know if this is really changing the output on the doc build, but when it's related to _running commands_ we started to tag the block with `console`, to differentiate it from `shell` scripts (which have its own highlight).

Reply:
It's the same for the below blocks.